flink-autoscaler-plugin-jdbc/src/test/java/org/apache/flink/autoscaler/jdbc/event/AbstractJdbcAutoscalerEventHandlerITCase.java

```diff
@@ -430,6 +446,70 @@ void testScalingEventWithParallelismChange() throws Exception {
                         });
     }
 
+    @MethodSource("getExpiredEventHandlersCaseMatrix")
+    @ParameterizedTest
+    void testCleanExpiredEvents(
+            int expiredRecordsNum, Duration eventHandlerTtl, int unexpiredRecordsNum)
+            throws Exception {
+        try (Connection con = getConnection();
+                PreparedStatement ps =
+                        con.prepareStatement("delete from t_flink_autoscaler_event_handler")) {
+            ps.execute();
+        }
```

Comment:
Did you find the `t_flink_autoscaler_event_handler` table has data?

Reply:
Here would be some data resulted by the parameterized test running due to the shared database instance.

Reply:
Actually, PostgreSQLExtension and MySQLExtension have `afterEach`, it cleans all data.

But they missed `t_flink_autoscaler_event_handler` table. I submitted https://github.com/apache/flink-kubernetes-operator/pull/866 to fix it.